src/server/app.ts

```diff
@@ -1,9 +1,23 @@
-var app = require('express')();
-var http = require('http').createServer(app);
-var io = require('socket.io')(http, {cookie: false, serveClient: false});
+const express = require('express');
+const https = require('https');
+const fs = require('fs');
+const socket = require('socket.io');
 import pos from '../pos';
 import windowManager from "../classes/window-manager";
 import posHandler from './poshandler';
+
+const app = express();
+
+const crtFolder: string = process.env.NODE_ENV === "production" ?
```

Comment:
These paths are hard to understand, can you find a way to make them more readable?